### PR TITLE
hotfix: get payments terms from cost center

### PR DESCRIPTION
**Pull request by Rudge**
#### What problem is this solving?

It was found a bug in checkout payment where the payment method is showing even when disabled on cost center.

I found the request to _v/private/b2b-checkout-settings on checkout and it responded with the payment terms from the organization instead of the cost center.

#changingTooling

#### How to test it?

- Access the My Organization in My Account with an "Organization Admin" role;
- Click on the Cost Center; 
- In the "Payment Terms" section, deactivate a payment term;
- Access the checkout and go to Payment; the deactivated option is still available.

[Workspace](https://b2bteam1244--b2bstore005.myvtex.com/)

#### Describe alternatives you've considered, if any.

It was considered to disable the payment terms for the organization, but it would be impractical.

### CODE REVIEW

**Review by enzomerca — DISMISSED** [review]
Nice! Thanks for adding tests!

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "license": "UNLICENSED",
   "scripts": {
     "lint": "eslint --ext js,jsx,ts,tsx .",
-    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\""
+    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\"",
+    "test": "cd ./node && yarn test"
```
Comment (mairatma) [inline package.json:8]: I believe we can use the [`--cwd` arg from yarn](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-cwd) instead here:

```suggestion
    "test": "yarn test --cwd ./node"
```

**Review by mairatma — DISMISSED** [review]
Good fix! And thanks for adding tests as well! 👏

File: node/resolvers/Routes/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    beforeEach(async () => {
+      context = mockContext()
+      await index.settings(context)
+    })
+    it('should return payments terms from cost center', () => {
+      const { response } = context
+
+      expect(response.status).toEqual(200)
+      expect(
+        (response.body as any)?.paymentTerms.some(
```
[the remark this answers is not in the document]
Comment (Rudge) [reply node/resolvers/Routes/index.test.ts:78]: I didn't find the type for this `body`, so I defined any. What do you think to declare as `{ paymentTerms: [] }`?

[the remark this answers is not in the document]
Comment (mairatma) [reply node/resolvers/Routes/index.test.ts:78]: Yep, I'd just give it a name though. That's probably the response form the settings API, right? So something like

```
interface SettingsResponse {
  paymentTerms: PaymentTerm[]
}

```

**Review by mairatma — APPROVED** [review]

**Pull request merged**